src/sql/workbench/contrib/dashboard/browser/pages/serverDashboardPage.contribution.ts

```diff
@@ -77,7 +77,7 @@ const defaultVal = [
 	{
 		name: 'Tasks',
 		widget: {
-			'tasks-widget': ['newQuery', 'mssqlCluster.task.newNotebook', { name: 'restore', when: '!mssql:iscloud && mssql:engineedition != 11' }, 'configureDashboard']
```

Comment:
hmm, let's keep the original order.

Reply:
ok switched back to original order for both server and db tasks